README.md

```diff
@@ -80,48 +80,37 @@ fn example_function (app_handle: AppHandle) {
     tokio::spawn({
         println!(receiver.once::<String>().await)
     });
+    channel
 }
 ```
 
 ### Frontend
 
-Use the `receive, emit` functions to receive or send events to/from backend
+Use the `Receiver.receive, Sender.emit` functions to receive or send events to/from backend
 
 ```typescript
-import { receive, emit, Channel } from 'tauri-plugin-channel';
-import {listen, emit} from "tauri-plugin-channel-api"
-import type {Channel} from "tauri-plugin-channel-api"
+import {channel} from "tauri-plugin-channel-api"
+import type {Sender, Receiver} from "tauri-plugin-channel-api"
 
-let channel = await invoke<Channel>('example_function')
-receive(channel, (event) => console.log(event));
-emit(channel, "1");
+type BackendEvents = { Progress: number } 
+
+[sender, receiver]  = await channel('example_function')
+receiver.listen<BackendEvents>(event => console.log(JSON.stringify(event)));
```

Comment:
I recall that you said the an unlisten function is not needed, Am I wrong?

Reply:
Ok, let me clarify this point:
When _all rust senders_ are dropped an "Unlisten" message is sent and the _ts receiver_ unmount itself and there is no need to call receiver.unlisten()
Unfortunately there is no way (or at least I don't see one) to enforce that senders are going to be dropped and therefore I've made `receiver::unlisten` public